script/test/fixtures/buildargs/output-os-template.json

```diff
@@ -113,7 +113,8 @@
                 "resources": {}
               }
             ],
-            "restartPolicy": "Always"
+            "restartPolicy": "Always",
+	    "securityContext": {}
```

Comment:
all of the tests add `securityContext`, we should have an if statement in the code that ONLY adds securityContext if it's been supplied. no need for blank resources like this.